GameDefinition/PLAYING.md

```diff
@@ -527,6 +538,104 @@ influence the speed of HP attrition and if it outweighs the benefits
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointsman's outfit, press 'O' (if you want to see the pointsman's
+personal inventory outfit and manipulate items stored there) or 'i' (if
+you want to stash something in the shared inventory from the pointsman's
+outfit).
+To open the shared inventory stash, press 'I'.
+To open the whole team's belongings (personal and stashed), press 'T'.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press 'G'.
+
+To pick up an item from the ground to the pointsman's outfit, press 'g' or ','.
+Items that have some direct use (melee weapon or armor) will be equipped 
+in the personal equipment outfit while if it has no direct use or the equippment 
+outfit is full, the item will be moved to the shared inventory stash.
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and 'RET' buttons to get the item
+management menu. You may want to choose all items from the particular location
+by pressing '*'.
+
+To drop an item on the ground from the inventory (either stashed or from
+the personal outfit), press 'r'.
+To trigger the item's properties (inject an ampoule, activate a chip, eat
+a piece of food), press 't'.
+To throw (fling) the item somewhere, press 'f'.
+To put the item into the particular outfit (personal or shared), press 'O'
+(personal outfit) or 'i' (shared inventory stash).
+
+You may want to travel around the characters' inventories without leaving
+the inventory menu. To do that, you may press '<' or '>' to check different
+inventories: character's outfit, stashed inventory, objects on the ground,
+etc. You may also change pointsmen that are adjusting inventory by pressing
+the 'TAB' command.
+
+Note how lowercase item commands (stash item, equip item) place items into
+a particular item store, while uppercase item commands (manage Inventory,
+manage Outfit) open the management menu for a store. Once a store menu is
+opened, you can switch stores with `<` and `>`, so the multiple commands
+only determine the starting item store. Each store is accessible from
+the dashboard as well.
+
+Quite often, you will need to modify your environment somehow. There are
+numerous ways how to do that, so try and experiment!
+You may craft items, change the terrain, cook food, upgrade or change items,
+and many other things!
+
+For example, to craft items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the workshop station, drop
+the components, stand on them, press 'M', and point to the direction
+of the item-modifying tile. The components will be automatically consumed,
+and the resulting item will appear. 
+For example, you may want to craft a club from a wooden stick and several 
```

Comment:
This is now well split. The repeated "For example" is distracting, though (repetition is a powerful literary device).

Reply:
Well, a fresh eye can always see things better.